ledger/sandbox/src/test/suite/scala/platform/sandboxnext/TlsRevokedCertificatesIT.scala

```diff
@@ -0,0 +1,115 @@
+package com.daml.platform.sandboxnext
+
+import java.io.File
+
+import com.daml.bazeltools.BazelRunfiles.rlocation
+import com.daml.ledger.api.domain
+import com.daml.ledger.api.testing.utils.AkkaBeforeAndAfterAll
+import com.daml.ledger.api.testing.utils.SuiteResourceManagementAroundEach
+import com.daml.ledger.api.tls.TlsConfiguration
+import com.daml.ledger.client.LedgerClient
+import com.daml.ledger.client.configuration.CommandClientConfiguration
+import com.daml.ledger.client.configuration.LedgerClientConfiguration
+import com.daml.ledger.client.configuration.LedgerIdRequirement
+import com.daml.platform.common.LedgerIdMode
+import com.daml.platform.sandbox.config.SandboxConfig
+import io.netty.handler.ssl.SslContext
+import org.scalatest.AsyncWordSpec
+import org.scalatest.Matchers
+
+import scala.concurrent.Future
+
+class TlsRevokedCertificatesIT
+  extends AsyncWordSpec
+    with Matchers
+    with AkkaBeforeAndAfterAll
+    with SuiteResourceManagementAroundEach
+    with SandboxNextFixture
+    with OCSPResponderFixture
+{
+
+  val List(
+  serverCrt,
```

Comment:
I don't think there's much gain using this "notation" over writing:
```
val serverCrt = "server.crt"
val serverKey = "server.pem"
```

Reply:
I followed a convention from the codebase but, on second thought, I agree that it's not very clear. I'll change that.

Reply:
Fixed and added the flag to kvutils.